core/http/src/uri/uri.rs

```diff
@@ -40,10 +40,10 @@ impl<'a> Uri<'a> {
         };
 
         Uri {
-            uri: uri,
-            path: path,
-            query: query,
-            fragment: fragment,
+            uri,
+            path,
+            query,
+            fragment,
```

Comment:
Can we put all four of these on the same line, here and elsewhere that something like this happens?